bin/pontoon-to-webext.js

```diff
@@ -0,0 +1,165 @@
+#! /usr/bin/env node
+
+/* eslint-disable promise/avoid-new */
+
+const propertiesParser = require('properties-parser');
+const path = require('path');
+const FS = require('q-io/fs');
+const argv = require('minimist')(process.argv.slice(2));
+
+const Habitat = require('habitat');
+Habitat.load();
+
+const regexPlaceholders = /\{([A-Za-z0-9_@]*)\}/g;
+let supportedLocales = process.env.SUPPORTED_LOCALES || '*';
+
+const config = {
+  'dest': argv.dest || 'dist/_locales',
+  'src': argv.src || 'locales',
+  'default_locale': argv.locale || 'en-US'
+};
+
+function log(...args) {
+  console.log(...args);  // eslint-disable-line no-console
+}
+
+function error(...args) {
+  console.error(...args);  // eslint-disable-line no-console
+}
+
+function getListLocales() {
+  return new Promise((resolve, reject) => {
+    if (supportedLocales === '*') {
+      FS.listDirectoryTree(path.join(process.cwd(), config.src)).then((dirTree) => {
+        const localeList = [];
+
+        // Get rid of the top level, we're only interested with what's inside it
+        dirTree.splice(0,1);
+        dirTree.forEach((localeLocation) => {
+          // Get the locale code from the end of the path. We're expecting the structure of Pontoon's output here
+          const langcode = localeLocation.split(path.sep).slice(-1)[0];
```

Comment:
(To be clear, this is upstream code that I'm hoping Les will turn into an npm package. We're using copypasta because it's expedient.)

So, this line is getting the last directory's name off an absolute path; the `slice(-1)[0]` is effectively the same as doing a `pop()` off the end of the array.

```js
let localeLocation = '/Users/jhirsch/codez/github/mozilla-services-pageshot/locales/en-US'
localeLocation.split('/')
// Array [ "", "Users", "jhirsch", "codez", "github", "mozilla-services-pageshot", "locales", "en-US" ]
localeLocation.split('/').slice(-1)
// Array [ "en-US" ]
localeLocation.split('/').slice(-1)[0]
// "en-US"
```